### DISCUSSION TITLE
Connection Issue Artisan v2.4.6, HRS10, MacBook Pro

### DISCUSSION THREAD

**Discussion by Schlueckchen** [artisan-roaster-scope/artisan#654] [Q&A]
Hello all,

I am trying to connect my new MacBook Pro to my Has Garanti HRS10 Roaster using my MacBook Pro.
With my old MacBook everything is working fine but once I switch to my new MacBook I face connection issues.

- MacBook Pro wit macOS Big Sur 11.4
- Has Garanti Roaster HRS 10
- Artisan v2.4.6
- serial driver CP210x as referred here: https://artisan-scope.org/modbus_serial/#serial-driver-cheat-sheet

Maybe you can point me to some hints to solve it?

Thanks for taking some time to look at it. Your effort is highly appreciated.

Best Tim

<img width="511" alt="Bildschirmfoto 2021-07-30 um 14 21 05" src="https://user-images.githubusercontent.com/88197098/127661421-3481fc64-919b-4a46-81c8-d7591c26fdda.png">
<img width="1189" alt="Bildschirmfoto 2021-07-30 um 14 23 59" src="https://user-images.githubusercontent.com/88197098/127661451-da483863-3d4c-47a7-a1bf-2d976c2c1c85.png">
<img width="495" alt="Bildschirmfoto 2021-07-30 um 14 26 57" src="https://user-images.githubusercontent.com/88197098/127661459-86785188-f5f2-453d-91de-64261cd6414a.png">
<img width="402" alt="Bildschirmfoto 2021-07-30 um 14 27 38" src="https://user-images.githubusercontent.com/88197098/127661476-5bc6aee5-b3aa-460b-86cb-9d20df106a66.png">

**Comment by CarefreeBuzzBuzz** [comment]
Have you tried copying your settings file from your old machine to your new one?  

Michael Herbert 
CarefreeBuzzBuzz

**Reply by Schlueckchen** [reply]
Hi Michael, 

No special settings have been made on the old computer.
But yes we tried that, too.
Unfortunately it didn't solve the issue.

Best Tim

**Comment by MAKOMO** [comment]
Obviously Artisan cannot communicate with the machine at all. Which version of the CP210x driver do you have installed on this machine. Only the very latest version v6.0.1 of the driver is compatible with the macOS Big Sur.

Did you read the drivers installation notes?

```
	Mount the DMG file and double click on 
	Silicon Labs VCP Driver. 
	On MacOS 10.13 and later, the installation of the driver may be blocked. 
	To unblock, open the System Preferences Security & Privacy pane and
	unblock the system extension. See Apple Technical Note TN2459
	"User-Approved Kernel Extension Loading" for more information.
```

**Reply by Schlueckchen** [reply]
Hi,

we had the most recent version v6.0.1 installed and approved the security settings.
Unfortunately it didn't solve the issue.

Do you have other details or hints we could check?

Best Tim

**Comment by CarefreeBuzzBuzz** [comment]
So since you have saved a settings file.  Try a factory reset and see if it
works to read temps.  Then go from there if it does.

On Fri, Jul 30, 2021 at 8:38 AM Schlueckchen ***@***.***>
wrote:

> Hi,
>
> we had the most recent version v6.0.1 installed and approved the security
> settings.
> Unfortunately it didn't solve the issue.
>
> Do you have other details or hints we could check?
>
> Best Tim
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/artisan-roaster-scope/artisan/discussions/654#discussioncomment-1091901>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIW3J4EHHUPAVPNS66CXTZDT2LBOLANCNFSM5BIRY6HA>
> .
>
-- 

__________________
Michael Herbert, Head Roaster
*Order at www.carefreebuzzbuzz.com <http://www.carefreebuzzbuzz.com>*
Artisan RoasterScope <https://artisan-scope.org/>Team

**Comment by CarefreeBuzzBuzz** [comment]
Also after the reset run the machine configuration or enter the settings  manually from your roasters instructions.  

Michael Herbert 
CarefreeBuzzBuzz

**Reply by MAKOMO** [reply]
I assume that the driver is not correctly loaded or the driver is not recognizing the external hardware for some reason and thus there is no serial port object generated by macOS and Artisan has nothing to connect to (or you simply did not select the correct serial port in the MODBUS tab).

Did you compare the USB entries in the macOS System Report on both machines. Once the USB device is connected you should see an additional entry as the FTDI one below (just in your case one for the CP210x).

<img width="852" alt="Screenshot 2021-07-30 at 20 21 56" src="https://user-images.githubusercontent.com/13909011/127696488-c1dd48fc-3232-4bca-b224-7c89a67f3f8b.png">

**Reply by Schlueckchen** [reply]
Thank you for being so responsive and helpful.
I will try all your hints and provide some feedback afterwards.
